src/operations.rs

```diff
@@ -154,13 +155,11 @@ pub fn download_extract_sans_parent(
             .progress_chars("=> "),
     );
 
-    let foo = pb.wrap_read(DataReaderWrap(reader));
-
-    let tar = GzDecoder::new(foo);
+    let response_with_pb = pb.wrap_read(DataReaderWrap(reader));
 
+    let tar = GzDecoder::new(response_with_pb);
     let archive = Archive::new(tar);
-
-    unpack_sans_parent(archive, &target_path, levels_to_skip)
+    unpack_sans_parent(archive, target_path, levels_to_skip)
```

Comment:
I don't see any particular reason to transfer ownership here if a reference works just as well, but still happy to merge this part of the diff if you think it's better for stylistic reasons.

Reply:
I agree, I actually prefer a ref here. But also, it really doesn't matter very much, so I'll merge this as soon as the merge window opens again.

Reply:
Just to explain: I merely wanted the two copies of code calling `unpack_sans_parent` to match as closely as possible, so that future edits don't have to wonder "why are these different?". Since I had to decide with two options, I used what `cargo clippy` considers as "best practice".